client/app/src/accounts/account.controller.js

```diff
@@ -1654,7 +1675,7 @@
 
     function isBIP39 (mnemonic) {
       const bip39 = require('bip39')
-      let valid = bip39.validateMnemonic(mnemonic)
+      let valid = bip39.validateMnemonic(mnemonic) || bip39.validateMnemonic(mnemonic, bip39.wordlists[self.getWordlistLanguage()])
```

Comment:
Why not just `let valid = bip39.validateMnemonic(mnemonic, bip39.wordlists[self.getWordlistLanguage()])` ? Default is English, for other languages you'll run `validateMnemonic()` twice, which is not necessary.

Reply:
Hi @zillionn thanks for your review, I explained this particular behavior on the ark-mobile PR : 
"For wallet import, entering a passphrase in english will still work, even if we select a different language in "Wordlist language". (we might want to generate wallets in a specific language but import a previous wallet which was generated in english)"

But I think I could test 1st the mnemonic in the defined language (changing the order in the "or" condition). Because if the language defined is different than English, there's a higher probability that we would import in this language.

What do you think ? And does it make sense to you to validate in English too even if we defined a different wordlist language ?

Thanks!

Reply:
I see. It's not up to me, I was just curious.